lib/pr_data.js

```diff
@@ -90,6 +91,14 @@ export default class PRData {
     ]);
   }
 
+  async getLabels() {
```

Comment:
I was confused for a moment why this extra method is needed, given that labels are already requested with the PR, but that's because this isn't actually getting the _labels_, but the _labled events_ from the timeline. I think this should be renamed. (Would apply to the query and the property as well.)